src/systemctl/systemctl-show.c

```diff
@@ -1332,6 +1383,23 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
                                 fputc('\n', stdout);
                         }
                         return 1;
+                } else if (STR_IN_SET(name, "StateDirectoryQuota", "CacheDirectoryQuota", "LogsDirectoryQuota")) {
+                        uint64_t quota_absolute;
+                        uint32_t quota_scale;
+                        const char *quota_enforce;
+
+                        r = sd_bus_message_read(m, "(tus)", &quota_absolute, &quota_scale, &quota_enforce);
+                        if (r < 0)
+                                return r;
+
+                        if (!parse_boolean(quota_enforce))
```

Comment:
this is a bit sloppy. please check for unexpected errrs here and propagate the error. i.e. parse_boolean() can return 0, 1, or negative on parse error.